builder/vsphere/iso/builder.go

```diff
@@ -40,13 +40,19 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 		&common.StepConnect{
 			Config: &b.config.ConnectConfig,
 		},
-		&packerCommon.StepDownload{
-			Checksum:    b.config.ISOChecksum,
-			Description: "ISO",
-			Extension:   b.config.TargetExtension,
-			ResultKey:   "iso_path",
-			TargetPath:  b.config.TargetPath,
-			Url:         b.config.ISOUrls,
+		&common.StepDownload{
+			DownloadStep: &packerCommon.StepDownload{
+				Checksum:    b.config.ISOChecksum,
+				Description: "ISO",
+				Extension:   b.config.TargetExtension,
+				ResultKey:   "iso_path",
+				TargetPath:  b.config.TargetPath,
+				Url:         b.config.ISOUrls,
+			},
+			Url:       b.config.ISOUrls,
+			ResultKey: "iso_path",
+			Datastore: b.config.Datastore,
+			Host:      b.config.Host,
```

Comment:
Duplicate fields are for testing?

Reply:
good catch! That's right, I started out with the intention of referencing them directly on the step, but that would have required me adding a "Get" method for these fields to the internal-step-interface so I could actually test it. That seemed just a bit unnecessary, though if we end up needing access to the common step's fields in a similar way for another builder it might be worth implementing down the road.